static/js/hoc/withChannelTracker_test.js

```diff
@@ -1,28 +1,28 @@
 import ReactGA from "react-ga"
 import { assert } from "chai"
+import { mount } from "enzyme"
+import React from "react"
 
 import { makeChannel } from "../factories/channels"
 import { withChannelTracker } from "./withChannelTracker"
 import { shouldIf, TestPage } from "../lib/test_utils"
 import IntegrationTestHelper from "../util/integration_test_helper"
 
 describe("withTracker", () => {
-  let helper, render, gaGaStub, channel, WrappedPage
+  let helper, gaGaStub, channel
+
+  const WrappedPage = withChannelTracker(TestPage)
+
+  const render = (state, props) => {
```

Comment:
Should these have default arguments? The `props` one should if it's meant to replace `configureHOCRenderer` since it also has a default argument for `props`.

Reply:
sure, makes sense